src/buttons/flat-button-label.jsx

```diff
@@ -61,19 +71,16 @@ const FlatButtonLabel = React.createClass({
       style,
     } = this.props;
 
-    const contextKeys = this.constructor.getRelevantContextKeys(this.state.muiTheme);
+    const {
+      prepareStyles,
+    } = this.state.muiTheme;
 
-    const mergedRootStyles = this.mergeStyles({
-      position: 'relative',
-      paddingLeft: contextKeys.spacingDesktopGutterLess,
-      paddingRight: contextKeys.spacingDesktopGutterLess,
-    }, style);
+    const styles = getStyles(this.props, this.state);
 
     return (
-      <span style={this.prepareStyles(mergedRootStyles)}>{label}</span>
+      <span style={prepareStyles(styles.root, style)}>{label}</span>
```

Comment:
Oh geez, good catch! I actually had meant to enable a warning for that too so we could catch it similarly to `callOnce`.

I think I might do that but not commit it because it's probably going to complain a lot with all the mixin implementations often using prepareStyles with multiple arguments.